hoomd/md/PotentialPairGPU.h

```diff
@@ -208,6 +208,9 @@ void PotentialPairGPU<evaluator, gpu_cgpf>::computeForces(uint64_t timestep)
                                  this->m_exec_conf->dev_prop),
              this->m_params.data());
 
+    // energy and pressure corrections
+    this->computeTailCorrection();
```

Comment:
Is computing this correction in a GPU kernel hard to do? It would improve performance if this could be a part of the GPU compute forces kernel.

Reply:
In NVT simulations, this PR as is poses minimal performance issues. The added code will only trigger on timesteps when energies/pressures are logged. However, in NPT simulations, the pressure correction will trigger on every timestep.

The expensive part of the computation is determining the number density of each particle type in the system. After that, the computation is a very inexpensive computation (per type pair, not per particle). The goal in this PR is to get a functioning code. @tommy-waltmann you are welcome to contribute a GPU kernel in a separate PR to count the number of particles of each type. The alternative would be to somehow detect when the number of particles of each type might change and only flag the expensive computation when needed.